pex/third_party.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import importlib
+import os
+
+from .tracer import TRACER
+
+
+class LazyImport(object):
```

Comment:
A docstring explaining why this is necessary (or something at the usage site) would be good.

Reply:
Agreed, I'll take a pass at docs here explaining the problems unit tests present with imports.